lib/index-basic.js

```diff
@@ -0,0 +1,18 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var Plotly = require('./core');
```

Comment:
I'd vote :-1: on adding a comment here (in all the other index files) to avoid making them obsolete when we'll strip more things for the `./core`. 

I would prefer pointing users to the `dist/README.md` file where [`coreModules`](https://github.com/plotly/plotly.js/pull/740/files#diff-870bb7c55c3bae2ed6d6a4603faac9ebR13) are listed:

![image](https://cloud.githubusercontent.com/assets/6675409/17062629/bf7f96b4-5001-11e6-9455-e349f9847d0f.png)